regexes/device/mobiles.yml

```diff
@@ -3989,6 +3992,10 @@ Celkon:
       model: '2GB Xpress'
     - regex: 'CliQ2(?:[);/ ]|$)'
       model: 'CliQ 2'
+    - regex: 'UniQ(?:[);/ ]|$)'
```

Comment:
Are you sure those detections for `UniQ` are always correct? Wondering if they can't have another android version by installing a custom rom 🤔

Reply:
I didn't find any other versions of android with the above case-sensitive version

fixed it, added a version
Zoom UNIQ - android 6
Celkon UniQ  - android 7

and added new brand `New Balance`  model: NB RunIQ